server/test/unit/server/webservices/views/test_repositories.py

```diff
@@ -1425,7 +1425,7 @@ def test_post_with_expected_content(self, mock_crit, mock_tags, mock_regen):
         """
 
         mock_request = mock.MagicMock()
-        mock_request.body = json.dumps({'repo_criteria': {}})
+        mock_request.body = json.dumps({'repo_criteria': {}, 'parallel': True})
```

Comment:
You probably need to bring back the tests for the code that is being brought back here too.